src/async/net/FreeRandomPort.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { AsyncObject } = require('@cuties/cutie')
+const portfinder = require('portfinder')
+
+class FreeRandomPort extends AsyncObject {
```

Comment:
@Guseyn I think it can be moved to `test/` directory

Reply:
@g4s8 we need FreeRandomPort in build process, not only in tests